docs/sources/reference/api/docker_remote_api_v1.18.md

```diff
@@ -1093,6 +1093,11 @@ Status Codes:
 -   **200** – no error
 -   **500** – server error
 
+> **Note**:
+> When the `remote` parameter references a Dockerfile then the `dockerfile`
+> parameter will have effect and the contents of the Dockerfile are placed
+> into a file called `Dockerfile`.
+
```

Comment:
@duglin  Yeah, I caught that when I got to the command. Sorry, for the delay. You can probably lift that section on using a GIT repository from the command and drop it into the API's description if it isn't already there.  (Not sure how that blurb about clone would work..)